lib/models/repository-states/present.js

```diff
@@ -17,6 +17,7 @@ import Remote from '../remote';
 import RemoteSet from '../remote-set';
 import Commit from '../commit';
 import OperationStates from '../operation-states';
+import {addEvent} from '../../reporter-proxy';
```

Comment:
one thing I worry about is, we currently have some metrics tracking in git-shell-out-strategy.  Which is basically as close to the "back end" as you can go.  We also have tracking on the UI layer, which is useful for seeing how many users are clicking specific buttons and whatnot.  If we are adding instrumentation to the middle layers of the application, how do we ensure we're not double counting events? Would it make more sense to move this instrumentation down a layer to keep it all in one place?